### PR TITLE
[stm32] i2c irq priority can be set

### PR DESCRIPTION
**Pull request by victorandrehc**
Add the possibility to set the i2c irq priority.

### CODE REVIEW

File: src/modm/platform/i2c/stm32-extended/i2c_master.hpp.in [hunk 1]
```diff
@@ -99,12 +99,12 @@ public:
 	 */
 	template<class SystemClock, baudrate_t baudrate=kBd(100), percent_t tolerance=pct(5)>
 	static void
-	initialize()
+	initialize(uint32_t isrPriority = 10u)
```
Comment (salkinium) [inline src/modm/platform/i2c/stm32-extended/i2c_master.hpp.in]:
```suggestion
	initialize(uint8_t isrPriority = 10)
```
Plus all other uses. Hardware is limited to 8-bit anyways.

**Review by salkinium — CHANGES_REQUESTED** [review]
Yes, but with 8-bit ;-p

File: src/modm/platform/i2c/stm32-extended/i2c_master.hpp.in (hunk 1 above)
Comment (victorandrehc) [reply src/modm/platform/i2c/stm32-extended/i2c_master.hpp.in]: I have changed it to be a byte only

**Review by salkinium — APPROVED** [review]
Thanks!

**Pull request merged**